src/_includes/feature_lists/cloud.njk

```diff
@@ -1,224 +1,121 @@
 {%
     set sections = [{
         label: "Features",
+        hosting: "cloud",
         rows: [{
-            id: 'hosting',
+            id: "hosting",
             label: "Hosting",
-            values: ['Self-Managed', 'FlowForge or Self-Managed', 'FlowForge or Self-Managed'],
+            values: ['FlowForge Cloud', 'FlowForge Cloud', 'FlowForge Cloud'],
             info: "<p>FlowForge can be deployed using Docker, Kubernetes or run the core FlowForge platform on your local machine directly.</p><p>FlowForge Cloud provides you with a browser-based, secure and cloud-based platform, with all of the FlowForge Premium features included and managed by us to reduce your operational overheads.</p>"
         }, {
-            id: 'multiple-projects',
-            label: "Multiple Applications",
-            values: ['check', 'check', 'check'],
-            info: "<p>Scale your Node-RED infrastructure with as many Node-RED instances as you need.</p><p>In FlowForge, a Project manages your instance of Node-RED, attached devices, context, environment variables, and version history.</p>"
+            id: "team-members",
+            label: "Team Members",
+            values: ['2', '20', 'Unlimited'],
+            info: "<p>Tooltip info here</p>"
```

Comment:
Hi @iskerrett, there are a few features here and in the self-hosted file that still have this placeholder text. Is it going to be replaced, or should it be deleted entirely?